src/translations/en.json

```diff
@@ -789,66 +789,6 @@
-  "StripeBankAccountTokenInputField.bsb.placeholder": "Type in BSB…",
-  "StripeBankAccountTokenInputField.bsb.required": "BSB is required.",
-  "StripeBankAccountTokenInputField.clabe.inline": "CLABE",
-  "StripeBankAccountTokenInputField.clabe.label": "CLABE",
-  "StripeBankAccountTokenInputField.clabe.placeholder": "Type in CLABE…",
-  "StripeBankAccountTokenInputField.clabe.required": "CLABE is required.",
-  "StripeBankAccountTokenInputField.clearingCode.inline": "clearing code",
-  "StripeBankAccountTokenInputField.clearingCode.label": "Clearing code",
-  "StripeBankAccountTokenInputField.clearingCode.placeholder": "Type in clearing code…",
-  "StripeBankAccountTokenInputField.clearingCode.required": "Clearing code is required.",
-  "StripeBankAccountTokenInputField.genericStripeError": "Couldn't connect account number. Please double-check that your {inputs} are valid in {country}",
-  "StripeBankAccountTokenInputField.genericStripeErrorIban": "Couldn't connect account number. Please double-check that your account number is valid in {country}",
-  "StripeBankAccountTokenInputField.iban.inline": "IBAN",
-  "StripeBankAccountTokenInputField.iban.label": "Bank account number (IBAN)",
-  "StripeBankAccountTokenInputField.iban.placeholder": "DE89 3704 0044 0532 0130 00",
-  "StripeBankAccountTokenInputField.iban.required": "Bank account number (IBAN) is required.",
-  "StripeBankAccountTokenInputField.institutionNumber.inline": "institution number",
-  "StripeBankAccountTokenInputField.institutionNumber.label": "Institution number",
-  "StripeBankAccountTokenInputField.institutionNumber.placeholder": "Type in institution number…",
-  "StripeBankAccountTokenInputField.institutionNumber.required": "Institution number is required.",
-  "StripeBankAccountTokenInputField.routingNumber.inline": "routing number",
-  "StripeBankAccountTokenInputField.routingNumber.label": "Routing number",
-  "StripeBankAccountTokenInputField.routingNumber.placeholder": "Type in routing number…",
-  "StripeBankAccountTokenInputField.routingNumber.required": "Routing number is required.",
-  "StripeBankAccountTokenInputField.sortCode.inline": "sort code",
-  "StripeBankAccountTokenInputField.sortCode.label": "Sort code",
-  "StripeBankAccountTokenInputField.sortCode.placeholder": "Type in sort code…",
-  "StripeBankAccountTokenInputField.sortCode.required": "Sort code is required.",
-  "StripeBankAccountTokenInputField.transitNumber.inline": "transit number",
-  "StripeBankAccountTokenInputField.transitNumber.label": "Transit number",
-  "StripeBankAccountTokenInputField.transitNumber.placeholder": "Type in transit number…",
-  "StripeBankAccountTokenInputField.transitNumber.required": "Transit number is required.",
-  "StripeBankAccountTokenInputField.unsupportedCountry": "Country not supported: {country}",
   "StripeConnectAccountForm.accountTypeTitle": "Account type",
   "StripeConnectAccountForm.bankAccountLabel": "Bank account",
```

Comment:
I guess we don't need this.

Reply:
@SariSaar I was wondering if the key "StripeConnectAccountForm.bankAccountLabel" is in use anywhere since the input field was removed. ...but I guess this is still shown because we show "stripeBankAccountLastDigits".

Reply:
Yep, it's shown here

<img width="450" alt="Screenshot 2024-10-16 at 16 53 40" src="https://github.com/user-attachments/assets/33833d3a-8e8a-42c9-b437-db27db8a23bd">